src/middle_end/linearize.ml

```diff
@@ -176,12 +176,12 @@ let rec transl_imm (({exp_desc; exp_loc=loc; exp_env=env; _} as e) : expression)
   | TExpConstruct _ -> failwith "NYI: transl_imm: Construct"
 
 
-and bind_patts ?toplevel:(toplevel=false) (exp_id : Ident.t) (patts : pattern list) : anf_bind list =
+and bind_patts ?exported:(exported=false) (exp_id : Ident.t) (patts : pattern list) : anf_bind list =
   let postprocess_item cur acc =
     match cur with
     | None -> acc
     | Some(ident, (src, idx), extras) ->
-      let bind = if toplevel then
+      let bind = if exported then
           BLetGlobal(Nonrecursive, [ident, Comp.tuple_get (Int32.of_int idx) (Imm.id src)])
```

Comment:
The semantics of this have shifted from its original design...while this in practice does exactly what you want, I would love to see this renamed from `BLetGlobal` to `BLetExport`.